tox.ini

```diff
@@ -0,0 +1,22 @@
+[tox]
+envlist = {py27}-django{18,111}
```

Comment:
You shouldn't need the `{}` around a single var, this can be `py27-django{18,111}`